omnigan/trainer.py

```diff
@@ -240,6 +243,7 @@ def setup(self):
         self.output_size = self.latent_shape[0] * 2 ** self.opts.gen.t.spade_n_up
         self.G.set_translation_decoder(self.latent_shape, self.device)
         self.D = get_dis(self.opts, verbose=self.verbose).to(self.device)
+        self.ADVENT_D = get_fc_discriminator().to(self.device)
```

Comment:
I liked to have all discriminators in `self.D`, and choosing specific keys to reach different models for different tasks. It made things like saving and sending to device a single straightforward op for all of them at once. I also think it makes the code cleaner if we decide to change advent for some reason.

What do you think @melisandeteng @51N84D @alexrey88 @sashavor ?

I mean that the advent discriminator could be a key in `OmniDiscriminator`. What do you people think?

Reply:
Okay, I'll fix that.

Reply:
Definitely agree. The code shouldn't be built around ADVENT, but rather ADVENT should just be a loss function / mode that we train the mask decoder with.